### PR TITLE
Add daily cron for elastic-agent

### PR DESCRIPTION
**Pull request by rowlandgeoff**
Adds the [daily cron that was manually configured](https://buildkite.com/elastic/elastic-agent/settings/schedules/621c1260-ba0a-4ab1-874e-0fe8b2a2797b) to the yml config.

### CODE REVIEW

File: .buildkite/integration.pipeline.yml [hunk 1]
```diff
@@ -95,7 +95,6 @@ steps:
     command: |
       buildkite-agent artifact download build/distributions/** . --step 'package-it'
       .buildkite/scripts/steps/beats_tests.sh
-    # if: "build.env('CRON') == 'yes'"
```
Comment (pazone) [inline .buildkite/integration.pipeline.yml:98]: Let's uncomment this line

Comment (rowlandgeoff) [reply .buildkite/integration.pipeline.yml:98]: Agent team, can you confirm the behavior you want? Currently the serverless beats tests are running with every single build, and it's been like that for 4 months. Do you only want the serverless beats tests running once per day?

Comment (rowlandgeoff) [reply .buildkite/integration.pipeline.yml:98]: I found the history. It looks like @pchila intentionally enabled them to run with every build - https://github.com/elastic/elastic-agent/pull/4501

**Review by blakerouse — APPROVED** [review]

**Comment by rowlandgeoff** [comment]
I have removed the manually configured cron at https://buildkite.com/elastic/elastic-agent/settings/schedules

File: .buildkite/integration.pipeline.yml (hunk 1 above)
Comment (pchila) [reply .buildkite/integration.pipeline.yml:98]: Sorry, just found this PR via its backport but I can still add a bit of context:

This was supposed to be temporary to make sure that agent/beats changes on PRs would not break serverless beats tests.

The whole step was supposed to be moved to the beats CI pipeline at some point but here it still is 6 months ago.

@pierrehilbert @jlind23 do we have an idea when these tests can be moved off the agent pipeline and into beats CI?

Comment (jlind23) [reply .buildkite/integration.pipeline.yml:98]: We were "only" waiting for buildkite to be available in the beats repository so I think we should be able to do it by now.

Comment (rowlandgeoff) [reply .buildkite/integration.pipeline.yml:98]: Tracking with https://github.com/elastic/ingest-dev/issues/4128, currently set for sprint 37

**Pull request merged**